wire-ios-sync-engine/Source/UserSession/ZMUserSession/ZMUserSession.swift

```diff
@@ -878,6 +879,24 @@ public final class ZMUserSession: NSObject {
 
     // MARK: - Trigger syncing
 
+    func triggerSyncsIfNeeded() async {
```

Comment:
thought: non-blocking, but I wonder if we need this `triggerSyncsIfNeeded` compared to `triggerSync` or maybe just think when to use one or the other

Reply:
We should probably jump on a call to discuss.